app/controllers/publish_document_controller.rb

```diff
@@ -8,6 +8,10 @@ def confirmation
   def publish
     document = Document.find_by_param(params[:id])
     review_state = params[:self_declared_review_state] == "has-been-reviewed" ? "reviewed" : "published_without_review"
+
+    images = [document.images, document.lead_image].flatten
```

Comment:
Maybe this can go into `DocumentPublishingService`?

Reply:
Makes sense 👍